src/schedule/ll_schedule.c

```diff
@@ -162,9 +170,12 @@ static void schedule_ll_tasks_run(void *data)
 {
 	struct ll_schedule_data *sch = data;
 	uint32_t num_clients = 0;
-	uint64_t next_tick;
 	uint32_t flags;
 
+	tr_dbg(&ll_tr, "timer interrupt on core %d, at %u, previous next_tick %u",
+	       cpu_get_id(), (unsigned int)platform_timer_get(timer_get()),
```

Comment:
> trace prints the current core anyway

this will be very verbose, that's why I use _debug(). the current core information is helpful for multi-core scenarios debugging.